roles/ffh.mesh_wireguard/templates/vxlan.conf.j2

```diff
@@ -0,0 +1,7 @@
+[Match]
+Name={{ vxlan_iface }}-{{ '%02d' % domain.id | int }}
```

Comment:
Maybe name it without a dash like the other devices

Reply:
I'm indifferent.

Reply:
I like it more with dashes, it support readability